sys/filesys_windows.go

```diff
@@ -311,9 +311,10 @@ func cleanupWCOWLayer(layerPath string) error {
 		HomeDir: filepath.Dir(layerPath),
 	}
 
-	// ERROR_DEV_NOT_EXIST is returned if the layer is not currently prepared.
+	// ERROR_DEV_NOT_EXIST is returned if the layer is not currently prepared or activated.
+	// ERROR_FLT_INSTANCE_NOT_FOUND is returned if the layer is currently activated but not prepared.
 	if err := hcsshim.UnprepareLayer(info, filepath.Base(layerPath)); err != nil {
-		if hcserror, ok := err.(*hcsshim.HcsError); !ok || hcserror.Err != windows.ERROR_DEV_NOT_EXIST {
+		if hcserror, ok := err.(*hcsshim.HcsError); !ok || (hcserror.Err != windows.ERROR_DEV_NOT_EXIST && hcserror.Err != syscall.Errno(windows.ERROR_FLT_INSTANCE_NOT_FOUND)) {
```

Comment:
I'm curious why `windows.ERROR_DEV_NOT_EXIST` doesn't get a case, but `windows.ERROR_FLT_INSTANCE_NOT_FOUND` does.

Reply:
I don't understand why they are declared this way, but one is a `syscall.Errno`, and one is a `Handle`.

https://pkg.go.dev/golang.org/x/sys/windows#ERROR_DEV_NOT_EXIST
```
ERROR_DEV_NOT_EXIST                                                       syscall.Errno = 55
```

https://pkg.go.dev/golang.org/x/sys/windows#ERROR_FLT_INSTANCE_NOT_FOUND
```
ERROR_FLT_INSTANCE_NOT_FOUND                                              Handle        = 0x801F0015
```

Reply:
`Handle` and `syscall.Errno` both appear to be a `uintptr` behind the scenes, so there shouldn't be any loss of precision here.

Reply:
My guess is `ERROR_FLT_INSTANCE_NOT_FOUND` is technically an HRESULT, so doesn't have the same semantics implied by the `syscall.Errno` type, so they used a `Handle` instead. For purposes of comparing the actual error codes I think the cast is fine, though.

Reply:
`Handle` is an interesting type-choice for a HRESULT, but yeah, we can just live with it. Thank you for indulging my curiosity.